Engine/ac/string.cpp

```diff
@@ -358,16 +377,17 @@ void * String_Split(const char *thisString, const char *separator)
         const char* tk_end = found_cstr;
         const char* nonSpaceBack = tk_end;
 
-        nonSpaceFront = TrimBack(nonSpaceFront, tk_end);
-
-        if (nonSpaceFront != tk_end) {
-            nonSpaceBack = TrimFront(tk_front, nonSpaceBack);
+        if(trim) {
+            nonSpaceFront = TrimBack(nonSpaceFront, tk_end);
+            if (nonSpaceFront != tk_end) {
+                nonSpaceBack = TrimFront(tk_front, nonSpaceBack);
+            }
         }
 
         len = nonSpaceBack - nonSpaceFront;
 
         ptr = found_cstr + seplen;
-        if(len == 0) continue;
+        if(removeEmpty && len == 0) continue;
```

Comment:
Please, keep 1 space between `if` and opening bracket, and place one-line statement on the next line under the `if` (according to our code style).

Reply:
I did a commit that I think fix the style issues. I want to squash all Engine commits in one because I think it makes it easier to read in history if that is alright.